config/client.js

```diff
@@ -1,7 +1,14 @@
-const Discord = require('discord.js');
+const {Client, Intents} = require('discord.js');
 
-const client = new Discord.Client({
+const client = new Client({
   partials: ['MESSAGE', 'CHANNEL', 'REACTION'],
+  intents: [
```

Comment:
I'm curious as to whether we need these partials anymore... 🤔 
Guess we'll see!

Reply:
I wasn't sure if we did, but I left them there in case they still worked in v13. We can definitely remove them later if they're not needed!